installing_on_vmware_vsphere/_topics/additional_configuration.md

```diff
@@ -18,8 +18,8 @@ performance:
     `vm.swappiness` setting to `1` in the `tuned.conf` file from the
     default of `vm.swappiness = 30`.
 
-  - Use the `noop` scheduler instead. See the [VMware
-    documentation](https://kb.vmware.com/s/article/2011861) for more
+  - Use the `noop` scheduler instead. For more information, please refer to “Deploying an OVF or OVA Template” topic in the [VMware
+    documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html) for more
```

Comment:
The link you've updated to doesn't actually have the documentation of the `noop` scheduler.  Were you looking for a more recent version?

Reply:
@agrare Can you comment here?  Is this even needed anymore?

Reply:
The only info from VMware on the kernel scheduler is https://kb.vmware.com/s/article/2011861 which is what this is being changed from, not sure why.

This specific link might not be necessary because it instructs you to "To implement this change, see the documentation for your Linux distribution." which is what we link to down below for RHEL